packages/vite/src/node/server/moduleGraph.ts

```diff
@@ -139,6 +140,11 @@ export class ModuleGraph {
     mod.ssrTransformResult = null
     mod.ssrModule = null
     mod.ssrError = null
+
+    // Fix #3033
```

Comment:
Not sure if this is right, sorry if it's not the case but I was wondering why the issue number doesn't match.
```suggestion
    // Fix #3303
```